app/server/app/public/data/community/extreme-weather.json

```
@@ -0,0 +1,851 @@
+          "query": {
+            "outFields": ["Temp"]
+          }
+        }
+      ]
+    }
+  ],
+
+  "historicalDefaults": [
+    {
+      "id": "fire",
+      "label": "Fire",
+      "checked": false,
+      "disabled": false,
+      "layerId": "cmraScreeningLayer",
+      "status": "idle",
+      "text": "",
+      "queries": [
+        {
+          "query": {
+            "where": "GEOID LIKE '{HMW_COUNTY_FIPS}%'"
+          }
+        }
+      ],
+      "layerProperties": {
+        "blendMode": "multiply",
+        "title": "Fire",
+        "renderer": {
+          "type": "class-breaks",
+          "field": "HISTORIC_MAX_CONSECDD",
+          "classBreakInfos": [
+            {
+              "minValue": -9007199254740991,
+              "maxValue": 9007199254740991,
+              "symbol": {
+                "type": "simple-fill",
+                "color": [170, 170, 170, 255],
+                "outline": {
+                  "type": "simple-line",
+                  "color": [225, 229, 232, 107],
+                  "width": 0,
+                  "style": "solid"
+                },
+                "style": "solid"
+              }
+            }
+          ],
+          "visualVariables": [
+            {
+              "type": "color",
+              "field": "HISTORIC_MAX_CONSECDD",
+              "stops": [
+                {
+                  "color": [255, 255, 178, 255],
+                  "value": 10
+                },
+                {
+                  "color": [254, 204, 92, 255],
+                  "value": 19
+                },
+                {
+                  "color": [253, 141, 60, 255],
+                  "value": 28
+                },
+                {
+                  "color": [240, 59, 32, 255],
+                  "value": 37
+                },
+                {
+                  "color": [189, 0, 38, 255],
+                  "value": 46
+                }
+              ]
+            },
+            {
+              "type": "size",
+              "valueExpression": "$view.scale",
+              "stops": [
+                {
+                  "size": 0.042905885932572586,
+                  "value": 1066256
+                },
+                {
+                  "size": 0.021452942966286293,
+                  "value": 3332050
+                },
+                {
+                  "size": 0.010726471483143147,
+                  "value": 13328201
+                },
+                {
+                  "size": 0,
+                  "value": 26656402
+                }
+              ],
+              "target": "outline"
+            }
+          ]
+        },
+        "popupTemplate": {
+          "title": "{CountyName}, {StateAbbr}",
+          "content": [
+            {
+              "type": "text",
+              "text": "<p><span style=\"font-size:18px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Maximum number of consecutive dry days</strong></p>"
```

This font is a bit larger than size we normally use.
```suggestion
              "text": "<p><span style=\"font-size:14px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Maximum number of consecutive dry days</strong></p>"
```